veashi-contracts/script/deploy-route.sh

```
@@ -0,0 +1,166 @@
+  esac
+done
+
+# ----------------------------
+# Validation
+# ----------------------------
+if [[ -z "$REPORTER_CHAIN" || -z "$ADAPTER_CHAIN" ]]; then
+  echo "❌ reporter-chain and adapter-chain are required"
+  exit 1
+fi
+
+if ! $USE_LZ && ! $USE_VEA && ! $USE_CCIP && !$USE_LIGHTBULB; then
```

Shellcheck (0.11.0)

[error] You are missing a required space after the !.

(SC1035)